angel_system/impls/detect_activities/two_stage/src/models/tbn_lightning.py

```diff
@@ -0,0 +1,186 @@
+"""###### Work in Progress ###### Implementation of Temporal Binding Networks.
+
+(official paper - "EPIC-Fusion: Audio-Visual Temporal Binding for Egocentric Action Recognition")
+(official implentation - http://github.com/ekazakos/temporal-binding-network)
+"""
+
+import pdb
+from typing import Any, Dict, List
+
+import torch
+from pytorch_lightning import LightningModule
+from torchmetrics import MaxMetric
+from torchmetrics.classification.accuracy import Accuracy
+
+
+class TBNLightning(LightningModule):
```

Comment:
I don't think this class get's used in the inference side of the world from what I can find. Otherwise, we should remove this, which also removes the use of a dependency (`pytorch_lightning`) that is not listed as a dependency of this package.